src/index.html

```diff
@@ -7,8 +7,77 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div>
+        <a href="#home" class="image">
+          <img src="images/logo.png" alt="moyo logo">
+        </a>
+      </div>
+
+      <div class="space"></div>
```

Comment:
```suggestion
```

you don't need this block at all! Insted of this, add to `header` flex and space between.
don't forget about `checklist` and using class names.

Reply:
Its sad. I thought its good idea for adaptive without meta queries